### ISSUE TITLE
Many duplicate schemas in the corpus

### ISSUE THREAD

**Issue by clairey-zx81** [sdbs-uni-p/usage-of-not#1]
Thanks for collecting all these JSON Schema!

While investigating the repository, *81987* JSON files are found, however there seems to be a lot of repeats, for instance file `pp_104.json`, which is JSON core schema meta-schema v5, seems to be replicated *1499* times (after JSON normalisation). 

Overall, there seems to be only *23619* unique JSON Schemas in the data.

Have you looked at this issue?

Thanks for your feedback,

**Comment by miniHive** [comment]
Yes, there are a lot of repeats - for our study on the "usage of not" in JSON Schema (https://link.springer.com/chapter/10.1007/978-3-030-89022-3_9), we removed the duplicates and ended up with 6K schemas that we believe to be unique.

In that paper, we write "The schema corpus contains a large share of near-duplicate schemas, with small variations in syntax. We performed duplicate elimination by comparing compact schema signatures, defined as a function that maps each JSON Schema keyword to the number of its occurrences in the schema (encoded as a vector of keyword counts); we assumed that two schemas with the same signature are, with high probability, versions of the same schema, and we retained just one."
